### ISSUE TITLE
Add Boyer

### ISSUE THREAD

**Issue by nocollier** [rubisco-sfa/IOMB-Data#7]
- [ ] Find current script and submit a PR adding it to IOMB-Data

**Comment by nocollier** [comment]
Is [this](https://cerweb.ifremer.fr/deboyer/mld/Surface_Mixed_Layer_Depth.php) the original source?

**Comment by nocollier** [comment]
To what do we compare this to in the models? I am seeing `mlotstmax` and `mlotstmin` in the CMIP6 ESGF archive. In your AR6 results, I see the variable was `omlmax`.

**Comment by nocollier** [comment]
The first link above is to the author's website where he has several versions of his product. At first I encoded the temperature threshold version simply because it was listed first. Subsequently, I discovered that the data is also available at SEANOE [here](https://www.seanoe.org/data/00806/91774/). This is nice because the data itself has a DOI, however, the only version I see is the density threshold version. This suggests to me that Boyer intends this to be the preferred version. Is there a reason to use one version over another?
